tests/clean.sh

```diff
@@ -1,22 +1,58 @@
 #!/bin/bash
 
-cf target -o pcfdev-org -s pcfdev-space
+cf api $CF_API_URL --skip-ssl-validation
```

Comment:
We might want to add a log entry trace such as "Start cleaning up potentially leaking resources from previous test executions. Warnings about missing resources should be ignored"